ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,37 @@
+### Steps to Reproduce
+
+1. 
+2. 
+3. 
+
+### Expected Behavior
+
+### Actual Behavior
+
+### Environment
+
+<!--
+1. From the Visual Studio main menu, select: Help > Visual Studio > Show Details > Copy Info (https://developer.xamarin.com/guides/cross-platform/troubleshooting/questions/version-logs/)
+
+2. Paste below this comment block.
+-->
+
+### Log File
```

Comment:
Build Logs ?

Most other log files are IDE and not useful (but time consuming to gather)